### DISCUSSION TITLE
How Restore 718 PCM 4 Back To Stock after applying MHI2_US_POG11_P5186_MU1476_PATCH.

### DISCUSSION THREAD

**Discussion by GAC1492** [Mr-MIBonk/M.I.B._More-Incredible-Bash#241] [Q&A]
Hi,
How do I use the backup/restore feature to revert the patched PCM back to stock (i.e., undo any changes I have made. I have the backup created by MIB.

Thanks

**Comment by harman-f** [comment]
There are restore functions available within M.I.B.

What is the reason to do this?

**Comment by GAC1492** [comment]
To remove the patch and the navigation option as the car is being sold and the new owner does not want a modified PCM.

**Comment by GAC1492** [comment]
Why is there a restore option on the MIB backup menu screen?

**Comment by GAC1492** [comment]
This screen shot below states that stock firmware can be restored via a user defined SWDL? Are there any directions on how to do this?
![image](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/135893347/bb84aafe-fda4-4150-9434-daaacef1e90e)

**Reply by harman-f** [reply]
There is a '`backup & restore' menu item.
However, functions within are not built to fully convert unit back.

Which functions did you enable with M.I.B?

**Comment by dchertousov** [comment]
Hi @harman-f !

I do have similar question. I have installed M.I.B. v3.6.0 and then applied all in one patch via menu.
If I ever want to sell the car and for this reason want to have unmodified unit software is there any way to remove all the changes?

I have in mind three options, but don't have any idea if any of them is viable:
1. Reflashing the unit with the stock firmware from red menu using SD card
2. Reflashing the unit with the stock firmware somewhere in the garage with external devices involved
3. Going to dealer with that
